ldes-server-port-fetch-rest/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/rest/eventstream/converters/EventStreamHttpConverter.java

```diff
@@ -39,8 +39,7 @@ public boolean canWrite(Class<?> clazz, MediaType mediaType) {
 
 	@Override
 	public List<MediaType> getSupportedMediaTypes() {
-		return List.of(MediaType.valueOf("text/turtle"), MediaType.valueOf("application/ld+json"),
-				MediaType.valueOf("application/n-quads"));
+		return List.of(MediaType.valueOf("*/*"));
```

Comment:
Is this the same as `MediaType.ALL`? If so, that's maybe an even cleaner way.

Reply:
good idea!